package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "tape test/*.test.js",
```

Comment:
The ```test``` command needs to be updated to the future new tests. At the moment it's running the old tests of the website. Maybe we can create a test folder inside the new website with just a test which check that the "/" endpoint is loaded correctly. We can test the status code 200 for example